keanu-python/examples/coal_mining.py

```diff
@@ -1,7 +1,7 @@
 import pandas as pd
-import keanu as kn
 import numpy as np
-
+from keanu.vertex import UniformInt, Exponential, DoubleIf, Poisson
+from keanu import Model
 
 class CoalMining():
```

Comment:
Nit: Do we get much value having this as a class rather than just a function? (I know this is nothing to do with the change at hand, but this feels like a leaking Java-ism? :)

Reply:
It's a curse

Reply:
Sounds good.